docs/config/MathJax.js

```diff
@@ -0,0 +1,20 @@
+window.MathJax = {
+    tex: {
+        tags: "ams",
+        packages: ['base', 'ams', 'bbox', 'color', 'physics']
+    },
+    options: {
+        ignoreHtmlClass: 'tex2jax_ignore',
+        processHtmlClass: 'tex2jax_process'
+    },
+    loader: {
+        load: ['[tex]/bbox', '[tex]/color', '[tex]/physics']
+    }
+};
+
+(function () {
+  var script = document.createElement('script');
+  script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js';
```

Comment:
Why are we changing from cloudflare to jsdelivr?

Reply:
Because that's what the MathJax instructions say to do :man_shrugging: